### PR TITLE
feat: add material-ui setup

### PR DESCRIPTION
**Pull request by frattezi**
Setup App entrypoint with Material UI styles

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -6,7 +6,13 @@
     "forceConsistentCasingInFileNames": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "lib": ["dom", "es2017"],
+    "lib": [
+      "dom",
+      "es2017"
+    ],
+    "noImplicitAny": true,
```
Comment (frattezi) [inline tsconfig.json:13]: Needed, typescript material UI recommended configurationons

File: pages/_app.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { createMuiTheme, responsiveFontSizes } from '@material-ui/core/styles';
+import ThemeProvider from '@material-ui/styles/ThemeProvider';
+import type { AppProps } from 'next/app';
+
+let theme = createMuiTheme();
+
+theme = responsiveFontSizes(theme);
```
Comment (marco-souza) [inline pages/_app.tsx]: can we have black themes with that?

File: pages/_app.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { createMuiTheme, responsiveFontSizes } from '@material-ui/core/styles';
```
Comment (marco-souza) [inline pages/_app.tsx]: Don't we need to import `css` to make it work? Is it magical like that?

**Review by marco-souza — APPROVED** [review]
lgtm

File: pages/_app.tsx (hunk 2 above)
Comment (frattezi) [reply pages/_app.tsx]: I think I wanna try something out.

File: pages/_app.tsx (hunk 3 above)
Comment (frattezi) [reply pages/_app.tsx]: simple as that

**Pull request merged**